src/Me/Modals/EditMentorDetails.js

```diff
@@ -0,0 +1,283 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+import { Modal } from './Modal';
+import FormField from '../components/FormField';
+import Input from '../components/Input';
+import Textarea from '../components/Textarea';
+import Select from '../components/Select';
+import Checkbox from '../components/Checkbox';
+import { desktop } from '../styles/shared/devices';
+import model from './model';
+import { fromMtoVM, fromVMtoM } from '../../helpers/user';
+
+const EditDetails = styled.div`
+  margin: 0 auto;
+  padding: 20px;
+  width: 100%;
+  max-width: 780px;
+`;
+
+const EditDetailsForm = styled.form`
+  display: flex;
+  width: 100%;
+`;
+
+const FormFields = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: stretch;
+  padding: 0 5px;
+  width: 100%;
+`;
+
+const ExtendedFormField = styled(FormField)`
+  flex: 1 1 100%;
+  max-width: 355px;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+
+  @media ${desktop} {
+    flex: 1 1 40%;
+  }
+`;
+
+const CustomFormField = styled.div`
+  display: 'flex';
+  flex-direction: 'column';
+  margin-bottom: 30px;
+`;
+
+const SubFieldContainer = styled.div``;
+
+const FormFieldName = styled.div`
+  margin-bottom: 2rem;
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 17px;
+`;
+
+const HelpText = styled.span`
+  display: block;
+  margin: 5px 0;
+  font-weight: normal;
+  font-size: inherit;
+  line-height: inherit;
+`;
+
+const FormErrors = styled.div`
+  /* padding: 0 10px; */
+`;
+
+function EditMentorDetails({
+  isModalOpen,
+  closeModal,
+  userDetails,
```

Comment:
Omit the avatar, otherwise the server will yell (see related issue: https://github.com/Coding-Coach/find-a-mentor-api/issues/143)

```suggestion
  userDetails: {avatar, ...details},
```
It comes with the next comment